lib/src/chewie_player.dart

```diff
@@ -495,11 +495,14 @@ class ChewieController extends ChangeNotifier {
   /// Defines a custom RoutePageBuilder for the fullscreen
   final ChewieRoutePageBuilder? routePageBuilder;
 
-  static ChewieController of(BuildContext context) {
+  static ChewieController of<ChewieControllerProvider>(BuildContext context) {
+    if (null is ChewieControllerProvider) {
+      return context.dependOnInheritedWidgetOfExactType<ChewieController>()!;
```

Comment:
`dependOnInheritedWidgetOfExactType` must depend on an `InheritedWidget`. `ChewieController` is not an `InheritedWidget`, it is a `ChangeNotifier`. This code will not work.